### DISCUSSION TITLE
Inconsistent UI components

### DISCUSSION THREAD

**Discussion by BenParizek** [craftcms/commerce#2806] [General]
While I won't call these inconsistencies a bug, it would be nice if the UI patterns on the settings pages in Craft and Commerce were consistent. Here are just 5 minutes worth of observations after clicking through the settings for Commerce 4:

## /admin/commerce/settings/ordersettings

![CleanShot 2022-05-06 at 11 27 00@2x](https://user-images.githubusercontent.com/309735/167163973-4f9e81f1-4128-46eb-928e-ee77cee89d50.png)

## /admin/commerce/settings/subscriptions/fields

![CleanShot 2022-05-06 at 11 27 39@2x](https://user-images.githubusercontent.com/309735/167164088-31484daf-0028-4412-9cae-dda5e30d3e9a.png)

## /admin/commerce/settings/gateways

![CleanShot 2022-05-06 at 11 28 25@2x](https://user-images.githubusercontent.com/309735/167164257-9bb64ca9-3483-4ab2-a1fc-a9b2e80d211a.png)

## /admin/commerce/settings/orderstatuses

![CleanShot 2022-05-06 at 11 29 21@2x](https://user-images.githubusercontent.com/309735/167164483-354ee652-c4b1-4f23-9e58-63c65b2e9d19.png)

**Comment by nfourtythree** [comment]
Hi @BenParizek 

Thank you for bringing these to our attention, I have created an issue and we will get fixes out.

Issue: #2808 

Thanks!
